naucse_render/course.py

```diff
@@ -13,7 +13,7 @@
 from .encode import encode_for_json, API_VERSION
 
 
-def get_course(course_slug: str = '', *, path='.', version=None):
+def get_course(course_slug: str = None, *, path='.', version=None):
```

Comment:
I am not good at type hints, but should str be wrapped in Optional now?

Reply:
That depends on the type checker. I write for humans, and consider this understandable.